microsoft-edge/devtools-guide-chromium/accessibility/emulate-vision-deficiencies.md

```diff
@@ -24,43 +24,31 @@ To better meet the needs of your users with [color vision deficiency](https://ww
 <!-- ====================================================================== -->
 ## Open the Rendering tool
 
-To simulate a vision deficiency being applied for your web product, open the [Rendering Tools](../rendering-tools/index.md).
+To simulate how a person with a vision deficiency would see your web page, open the [Rendering Tools](../rendering-tools/index.md).
 
-1.  To open the Rendering tool, select the `...` menu item in the toolbar.
-1.  Select **More tools**.
-1.  Select **Rendering**.
+1.  Click **Customize and control DevTools** (`...`) in the toolbar.
+1.  Go to **More tools** > **Rendering** to open the **Rendering** panel in the drawer.
 
-    :::image type="content" source="../media/getting-to-the-rendering-tools.msft.png" alt-text="Opening the Rendering tool." lightbox="../media/getting-to-the-rendering-tools.msft.png":::
+    ![Opening the Rendering panel from the More tools menu](../media/getting-to-the-rendering-tools.msft.png)
```

Comment:
If you make this image narrower than 1200px, by dragging the window to be smaller before capturing, the result will be crisper.  There's too much info in this screenshot, so the result is small, blurry UI elements.  I tried implementing this, but the borders weren't matching the other images, on my machine.

Reply:
try per DCG:
```suggestion
> [!div class="mx-imgBorder"]
> ![Opening the Rendering panel from the More tools menu](../media/getting-to-the-rendering-tools.msft.png)
```

Reply:
No I don't think `>` is required.